### PR TITLE
Cross thing trigger

### PR DESCRIPTION
**Pull request by yongpingchen**
When create command trigger, it is possible to set another thing as command target. 
In the trigger new/update UI, add commandTarget text filed, which should be filled with thing id of command target. 

**NOTE**: need to wait for thing-if-JS to be released.

### CODE REVIEW

File: src/app/pages/trigger/trigger.component.ts [hunk 1]
```diff
@@ -31,12 +33,15 @@ export class TriggerComponent {
   @ViewChild('enabledModal') public enabledModal: ModalDirective;
   public alerts: Array<Object> = [
   ];
+  commandTarget = null;
```
Comment (rizakii) [inline src/app/pages/trigger/trigger.component.ts]: remove semicolon

File: src/app/pages/trigger/trigger.service.ts [hunk 2]
```diff
@@ -105,15 +105,20 @@ export class TriggerService {
     }
   }
 
-  saveCommandTrigger(conditionPower: boolean, rgb: RGB, power: boolean, triggerID?: string): Promise<any> {
+  saveCommandTrigger(conditionPower: boolean, rgb: RGB, power: boolean, triggerID?: string, commandTarget?: string): Promise<any> {
```
Comment (rizakii) [inline src/app/pages/trigger/trigger.service.ts]: the argument is already to long, would you define an object as an argument.

**Comment by yongpingchen** [comment]
@rizakii fixed. Please confirm

**Comment by rizakii** [comment]
LGTM!

**Pull request merged**